code/game/machinery/cryopod.dm

```diff
@@ -9,25 +9,24 @@
 
 //Main cryopod console.
 
+var/global/list/frozen_crew = list()
```

Comment:
Why did you make this a global?

Reply:
So that any cryogenics computer you examine has the logs of who got stored. It could easily be moved into the computer only, if needed